### DISCUSSION TITLE
Renderer process crashed (status -2147483645) on windows

### DISCUSSION THREAD

**Discussion by murryguy** [qutebrowser/qutebrowser#8396] [Q&A]
**Version info**:
https://bpa.st/CS2Q

**Does the bug happen if you start with `--temp-basedir`?**:
NO

**Description**
I am in Windows, and yes, I installed this via winget. This error is the first one I got, and I always get it.

**How to reproduce**
This is affected by every website except DuckDuckGo.
1. Lanch qutebrowser
2. Go to a website (but not duckduckgo)
3. Wait for a few seconds

**Comment by The-Compiler** [comment]
Converted to a discussion because there is still nothing qutebrowser will be able to do about this.

A couple of ideas:

- See if there are any error messages if you launch in the terminal (but without `--temp-basedir`)
- Try quitting qutebrowser, removing `C:\Users\ottof\AppData\Local\qutebrowser\cache` and maybe `C:\Users\ottof\AppData\Roaming\qutebrowser\data` and see if that helps
- Try the [standalone release](https://github.com/qutebrowser/qutebrowser/releases/tag/v3.3.1) and see if that works any better, maybe the WinGet version is just broken. In any case, it's also outdated, try with v3.3.1 instead.
- Wait for v3.4.0 bundling Qt 6.8 and see if that changes anything.
